meson.build

```diff
@@ -2225,6 +2227,25 @@ if conf.get('HAVE_SYSV_COMPAT') == 1
                    install_dir : systemgeneratordir)
 endif
 
+if conf.get('ENABLE_XDG_AUTOSTART') == 1
+        executable('systemd-xdg-autostart-generator',
+                   [ 'src/xdg-autostart-generator/xdg-autostart-generator.c',
+                     'src/xdg-autostart-generator/xdg-autostart-service.c'],
+                   include_directories : includes,
+                   link_with : [libshared],
+                   install_rpath : rootlibexecdir,
+                   install : true,
+                   install_dir : usergeneratordir)
+
+        executable('systemd-xdg-autostart-condition',
```

Comment:
hmm, what's this? can you briefly explain this in the commit msg?

Reply:
I added a short paragraph to the commit message